server/src/main/java/io/spine/server/command/Command.java

```diff
@@ -62,12 +62,12 @@
  *
  * <h1>Command Transformation</h1>
  *
- * <p>Commanding methods may be used to transform an incoming command with one or more commands.
+ * <p>Commanding methods may be used to transform an incoming command into one or more commands.
  * In this case the first method parameter must extend {@linkplain io.spine.base.CommandMessage}.
  * The returning values must derive from the {@linkplain io.spine.base.CommandMessage} as well.
  *
- * <p>If a commanding method accepts a command type which is also handled by the application
- * by another method, a run-time error will occur. This also means that an application cannot have
+ * <p>If a commanding method accepts a command type which is also handled by another application
```

Comment:
Let's rephrase it in a different manner:

"... which is also handled by another method of the same application, a run-time error..."